question-servers/elements/pl_answer_panel.py

```diff
@@ -18,5 +18,8 @@ def parse(element_html, element_index, data):
 def grade(element_html, element_index, data):
     return data
 
-def testcase(element_html, element_index, data):
+def test(element_html, element_index, data):
     return data
+
+def file(element_html, element_index, data):
```

Comment:
That's reasonable. For consistency, why require any of them at all? Even render - if that's missing, freeform.js can assume the html is empty.